package.json

```diff
@@ -1,9 +1,7 @@
 {
   "scripts": {
-    "layer0": "layer0",
     "layer0:dev": "layer0 dev",
     "layer0:build": "layer0 build",
-    "layer0:deploy": "layer0 deploy layer0-docs",
```

Comment:
let's keep `layer0:deploy` and pull `deploy`. update the github workflow as needed